tensorflow_model_optimization/g3doc/_index.yaml

```diff
@@ -84,6 +84,12 @@ landing_page:
       - label: "Read on TensorFlow blog"
         path: https://blog.tensorflow.org/2020/04/quantization-aware-training-with-tensorflow-model-optimization-toolkit.html
 
+    - heading: "TensorFlow Model Optimization Toolkit — Collaborative Optimization API"
```

Comment:
Please put this item at the first in the list.

Reply:
Hi @rino20 Thanks for the review! Corrected.